qemu/tests/cfg/uefi_boot_from_device.cfg

```diff
@@ -0,0 +1,82 @@
+- uefi_boot_from_device:
+    virt_test_type = qemu
+    type = uefi_boot_from_device
+    only q35
+    only ovmf
+    image_boot = no
+    boot_menu = on
+    enable_sga = yes
+    boot_menu_key = "esc;down;down;kp_enter"
+    boot_menu_hint = "Boot Options"
+    images = "stg"
+    image_name_stg = "images/stg"
+    image_size_stg = 100M
+    force_create_image_stg = yes
+    remove_image_stg = yes
+    variants:
+        - cdrom:
+            cdroms = "test"
+            cdrom_test = "/tmp/test.iso"
+            boot_entry_info = "Booting UEFI QEMU .*(QEMU CD-ROM|DVD-ROM QM)"
+            start_vm = no
+            dev_name = cdrom
+            variants:
```

Comment:
what about move these variants out, like:
variants:
    - cdrom:
       ......
variants:
        - with_bootindex:
            bootindex_stg = 0
            cdrom:
                bootindex_stg = ''
                bootindex_test = 0
        - with_specify_device:
            cdrom:
                boot_dev = "UEFI QEMU (?:QEMU CD-ROM|DVD-ROM QM)"
         ......

Reply:
I think no need to update. It's more clearer. thanks.